latex/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+install(FILES pdfpc.sty
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/share/texmf/tex/latex/pdfpc
```

Comment:
This path causes trouble for sure on different systems.

Reply:
> This path causes trouble for sure on different systems.

Probably, yes. But I don't know what is a standard way to infer the proper location. BTW, in the previous comment you wanted to install the style file even without latex present. That for sure would be impossible (to know in advance the right directory)...

Reply:
The correct path with TeXLive would depend on the installation settings, the TeXLive version and on the fact whether it's vanilla TeXLive or a packaged version from a Linux distribution. With vanilla TeXLive 2019, it's
`/usr/local/texlive/2019/texmf-var/tex/latex/pdfpc/`
(note that packages not coming from TeXLive itself need to go to `texmf-var/` rather than `texmf-dist/`)

Note further that:
* a TeXLive upgrade will make the package unavailable again (as the path is changed)
* some people deliberately use different TeXLive versions (e.g. 2018 and 2019 in parallel)
* this all doesn't include MikTeX, where paths (and IIRC also the filename database update programs are completely different) and neither TeXLive on Windows and the Mac.